### DISCUSSION TITLE
New release

### DISCUSSION THREAD

**Discussion by korrat** [polarmutex/beancount-language-server#446] [General]
Hi, thanks for the awesome project!

Could you please cut a new release? The latest released version does not contain https://github.com/polarmutex/beancount-language-server/commit/1b685ed27af05e4950c3cb3845eb3a9048f3dadc yet, so anytime the language server starts, it leaves a `beancount-language-server.log` behind, which can get a bit annoying. Thank you very much!

Also, I've noticed that the last two versions, 1.3.2 and 1.3.3, were not published to crates.io. Was that intentional? If so, you might want to revise the installation instructions in the README, because `cargo install` is currently recommended.

**Comment by polarmutex** [comment]
sorry, thought that fix was already released. I will generate a release over the weekend
